### DISCUSSION TITLE
I just got a new Echo Show 10 and when I send TTS to it I get API error

### DISCUSSION THREAD

**Discussion by monsieurlatte** [alandtse/alexa_media_player#1231] [General]
Has anyone else come across this before?   All my other devices are still working with the same code.

**Comment by alandtse** [comment]
If you're getting an API error, you can try to create a [routine](https://github.com/custom-components/alexa_media_player/wiki/Developers%3A-Sequence-Discovery) to get the call that works using the app and file a bug report on this.
